src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.xml

```diff
@@ -210,6 +210,7 @@
             <argument /> <!-- Simple Authenticator -->
             <argument /> <!-- User Provider -->
             <argument /> <!-- Provider-shared Key -->
+            <argument>null</argument><!-- User Checker -->
```

Comment:
explicit `null` because while `SimpleFormFactory;;createAuthProvider()` does replace the argument, `SimplePreAuthenticationFactory::create()` doesn't

Reply:
Success tested on `4.0.7`
<img width="402" alt="screen shot 2018-04-03 at 3 56 47 pm" src="https://user-images.githubusercontent.com/2576509/38250510-a5a190e6-3757-11e8-9fc7-de40728b18c0.png">

<img width="889" alt="screen shot 2018-04-03 at 3 56 10 pm" src="https://user-images.githubusercontent.com/2576509/38250478-917dd020-3757-11e8-8cab-df16d62e1e9c.png">

Reply:
yes its works for me